modules/oadp-kopia-algorithms-benchmarking.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/overriding-kopia-algorithms.adoc          
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-kopia-algorithms-benchmarking_{context}"]
+= Benchmarking Kopia hashing, encryption, and splitter algorithms
+
+You can run Kopia commands to benchmark the hashing, encryption, and splitter algorithms. Based on the benchmarking results, it is possible to select the most suitable algorithm for your workload.
+
+.Prerequisites
+
+* You have installed the {oadp-short} Operator.
+* You have installed the Kopia client.
+* You have an application with persistent volumes running in a separate namespace.
+* You have run a backup of the application with CSI snapshots.
+
+.Procedure
+
+. Connect to the Kopia repository by running the following command:
```

Comment:
This is an example, of course this depends on a specific s3 storage options, entire kopia CLI subset of commands. However I am not sure we want to point to that, we should mention that this is an example ? E.g. disable-tls may not be used every single time?:

https://kopia.io/docs/reference/command-line/common/repository-connect-s3/